packages/libs/react-ui/src/components/Dialog/Dialog.stories.tsx

```diff
@@ -45,6 +45,13 @@ A Dialog is a type of modal that is used to display information or prompt the us
         type: { summary: 'string' },
       },
     },
+    size: {
+      control: {
+        type: 'radio',
+      },
+      options: ['compact', 'base', 'large'],
```

Comment:
I just put it here because it's readable but I don't like the naming for the 3 variants. What if we would add another size to this array which is smaller than compact? I think that we should be consistent and only use the compact naming if it's a more condensed look and basically the only alternative to the regular size. (so only when we have the default styles and have a smaller version)

For this situation I would suggest to use t shirt sizing. But perhaps check the other components how we named them to be consistent.

Reply:
I used the naming from the figma documentation from isa, I also thought we should use the normal sizes (or standard) but if they go and read the docs would be confusing. Ill ask isa to update and make a separate pr for that given he is 1 week off